public/static/js/spawned_hosts.js

```diff
@@ -484,7 +484,8 @@ mciModule.controller('SpawnedHostsCtrl', ['$scope', '$window', '$timeout', '$q',
       mciSpawnRestService.spawnHost(
         $scope.spawnInfo, {}, {
           success: function (resp) {
-            $window.location.reload();
+            window.location.href = "/spawn";
+            $scope.setResourceType("hosts");
```

Comment:
I think the lines after the refresh won't run.
If this is correct, an alternative could be to set it as a URL parameter.

Reply:
I did test this and the resource type is being set but I don't know how any of this works haha.

Reply:
I think the resource type gets [set to hosts automatically](https://github.com/evergreen-ci/evergreen/blob/98badaba32696a58d765e28a30b8c0b7cd9cda16/public/static/js/spawned_hosts.js#L53) when the page loads if the `resourcetype` parameter isn't in the URL.

Reply:
Oh interesting--okay so then I'm only going to make this change for hosts. I actually don't need to make it for volumes since only hosts can be spawned by tasks. Since this is going to look inconsistent without context I'll add a comment.

Reply:
it occurred to me that one nice thing about `$window.location.reload();` is that the host you had selected before you make a change is still selected after the reload. 
Is is possible to drop the distro_id and task_id and then call `$window.location.reload();`? Alternatively, could we set `$window.location.href` to `$window.location.href` minus the distro_id and task_id parameters?

Reply:
I'd rather not, I think modifying parameters explicitly is awkward and error prone, and the benefit of reload is not that great.